src/styles.css

```diff
@@ -39,46 +72,75 @@ input::placeholder {
   flex-direction: column;
 }
 
+.right {
+  display: flex;
+  flex-direction: column;
+  margin-top: 40px;
+}
+
 .heading {
   display: flex;
   flex-direction: row;
-  gap: 60px;
+  margin: 20px;
+  margin-left: 40px;
+  font-size: 22px;
+  transition: width 2s;
+}
+
+.heading h2 {
+  text-shadow: 1px 1px 2px rgb(238, 255, 0), 0 0 1em rgba(36, 2, 33, 0.479), 0 0 0.2em rgb(52, 52, 52);
+  background-color: rgba(2, 2, 2, 0.5);
+  line-height: 60px;
+  margin-top: 10px;
 }
 
 .scoreboard {
-  border: 3px solid black;
-  margin: 20px 0;
-  width: 70%;
+  border: 3px solid rgb(196, 37, 37);
+  margin: 20px;
+  width: 90%;
 }
```

Comment:
- [ ] there's a small issue when i refresh the scores i can not see all of them or scroll, kindly fix this issue

![image](https://user-images.githubusercontent.com/89282221/216090644-64c1bf29-6b4c-462b-917d-77b1b12dcc10.png)


**_i suggest updating this class :_**

```CSS
.scoreboard {
    border: 3px solid rgb(196, 37, 37);
    margin: 20px;
    width: 90%;
   /*Add max-height and set the overflow to auto */
    max-height: 220px;
    overflow: auto;
}
```

Reply:
Thank you for the feedback.